resources/views/global/mega_menu.blade.php

```diff
@@ -6,102 +6,114 @@
 $reports    = "/reports/";
 @endphp
 <nav class="navbar navbar-light bg-light navbar-expand-lg" id="myNavbar">
- <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#mainNav" aria-controls="mainNav" aria-expanded="false" aria-label="Toggle navigation">
-  <span class="navbar-toggler-icon"></span>
- </button>
- <a href="/" class="navbar-brand">Bluesky</a>
- <div class="collapse navbar-collapse" id="mainNav">
-  <ul class="navbar-nav ml-auto nav-fill">
-    
-    @if (Route::has('login'))
-            @auth
-            <li class="nav-item px-4">
-                <li class="nav-item dropdown">
-                    <a id="navbarDropdown" class="nav-link dropdown-toggle" href="#" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" v-pre>
-                        {{ Auth::user()->name }} <span class="caret"></span>
-                    </a>
-                    <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown">
-                        <a class="dropdown-item" href="{{ route('logout') }}"
-                            onclick="event.preventDefault();
-                                            document.getElementById('logout-form').submit();">
-                            {{ __('Logout') }}
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#mainNav" aria-controls="mainNav" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+    </button>
+    @if (!Auth::guest())
+        <a href="{{Auth::user()->home}}" class="navbar-brand">Bluesky</a>
+    @else
+        <a href="/" class="navbar-brand">Bluesky</a>
+    @endif
+    <div class="collapse navbar-collapse" id="mainNav">
+        <ul class="navbar-nav ml-auto nav-fill">
+            @if (Route::has('login'))
```

Comment:
Make login it's own route, extract from here into it's own blade template

Reply:
can you expand on this? This is only to show the login menu element on condition